python/analyzers/treeVariables.py

```diff
@@ -45,6 +45,13 @@ def default():
     Variable('puweight', lambda ev : ev.puWeight, type=float),
 ]
 
+# generic HNL reconstruction event variables
+hnlreco_vars = [
+    Variable('n_sMu', lambda ev : ev.n_sMu, type=int),
+    Variable('n_dSAMu', lambda ev : ev.n_sMu, type=int),
+    Variable('n_dimuon', lambda ev : ev.n_dimuon, type=int),
+]
+
```

Comment:
mmm, I don't like this too much.
These are event-specific variables, not so much single-HNL-specific.